v4l2/v4l2_codecbase.cpp

```diff
@@ -279,12 +281,13 @@ const char* V4l2CodecBase::IoctlCommandString(int command)
 }
 #endif
 
-int32_t V4l2CodecBase::ioctl(int command, void* arg)
+int32_t V4l2CodecBase::ioctl(uint64_t command, void* arg)
```

Comment:
do we have problem when we compile this in chrome. since chrome may define command type to int